erpnext_documentation/www/docs/user/manual/en/healthcare/patient_assessment.md

```diff
@@ -0,0 +1,28 @@
+<!-- add-breadcrumbs -->
+
+# Patient Assessment
+
+The Patient Assessments in Rehabilitation provide evidence to support the development of a rehabilitation plan, tailored to the client's needs, circumstances and goals. Healthcare Practitioners have to record assessments at regular intervals to track Patient Progress. There are various assessments that are conducted for the Patient like Motor Assessment Scale, Functional Mobility Scale, Stroke Rehabilitation Assessment of Movement (STREAM), etc. Patient Assessment DocType helps you capture these assessments at any point of time during the Therapy Plan.
+
+To create a Patient Assessment, go to:
+
+> Home > Healthcare > Rehabilitation and Physiotherapy > Patient Assessment
+
+## 1. How to Create a Patient Assessment
+
+1. Go to the Patient Assessment list, click on New.
+2. You can optionally select a Therapy Session after which the evaluation is being conducted. Details like Patient and Heathcare Practitioner will be fetched automatically.
+3. Select the Patient Assessment Template. The Assessment Description and all the Assessment Parameters will be fetched automatically. The scale for patient evaluation will be set in the "Score" field in child table.
```

Comment:
again, name the child table (don't use the word child table.. say in "table name" table)